decentraland/unity-explorer
# fix: loading bar progression

The loading bar of the loading screen was not correctly representing the values assigned from the load report. The reserved % for the minimum wait time was growing to 100%. It has been fixed to 20%, letting the 80% of the current process to be represented in the rest of the bar.
Also fixes the propagation of the progress values.

# How to test?

1. Start the app and go through the startup
2. Check that the loading bar progress as expected
3. Teleport to any other scene
4. Check that the loading bar progress as expected
5. Change the realm and check that the loading bar progress as expected
